addon-test-support/@ember/test-helpers/wait-until.js

```diff
@@ -1,6 +1,8 @@
 import { Promise } from 'rsvp';
+import { futureTick } from './-utils';
 
-import { nextTick } from './-utils';
+const TIMEOUTS = [0, 1, 2, 5, 7];
```

Comment:
7?

Reply:
Yeah, I added another entry. Does It seem bad?